db/migrate/20150129120103_add_default_to_category_ordinal.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultToCategoryOrdinal < ActiveRecord::Migration
+  def change
+    change_column :comfy_cms_categories, :ordinal, :integer, default: 999
```

Comment:
Don't take me seriously, this was fine. My emoji were reference to the `999` being the number you dial for police here :smile: Leave it as-is!